src/app/nav-bar/nav-bar.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="nav-bar w-100">
+    <evt-button *ngIf="thumbnailsButton" [iconRight]="{icon: 'th'}" additionalClasses="mr-1" [active]="thumbnailsPanelOpened$ | async"
+        (btnClick)="toggleThumbnailsPanel()" [title]="'thumbnails' | translate"></evt-button>
+    <evt-button *ngIf="viscollButton" [iconRight]="{icon: 'rainbow', rotate:270}" additionalClasses="mr-1" [active]="viscollPanelOpened$ | async"
+        (btnClick)="toggleViscollPanel()" [title]="'viscoll' | translate"></evt-button>
+    <ngx-slider *ngIf="pageSliderOptions$ | async as pageSliderOptions" class="ml-2 mr-2"
```

Comment:
If you drag the ngxslider pointer it doesn't change the page. Is the expected behavior?

Reply:
It does change. Can you check again? (Note that the page change is not yet implemented for critical edition!)